src/main/java/edu/cornell/kfs/module/purap/document/validation/impl/CuCreditMemoDocumentPreRules.java

```diff
@@ -14,17 +11,15 @@
 import org.kuali.kfs.module.purap.document.validation.impl.CreditMemoDocumentPreRules;
 import org.kuali.kfs.sys.KFSConstants;
 import org.kuali.kfs.sys.KFSPropertyConstants;
+import org.kuali.kfs.sys.businessobject.PaymentSourceWireTransfer;
 import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.kfs.sys.document.validation.impl.PaymentSourcePreRulesServiceImpl;
 
-import edu.cornell.kfs.module.purap.CUPurapConstants;
-import edu.cornell.kfs.module.purap.businessobject.CreditMemoWireTransfer;
-import edu.cornell.kfs.module.purap.document.CuVendorCreditMemoDocument;
 import edu.cornell.kfs.pdp.service.CuCheckStubService;
-import edu.cornell.kfs.sys.CUKFSKeyConstants;
-
 
 public class CuCreditMemoDocumentPreRules extends CreditMemoDocumentPreRules {
     
+    private PaymentSourcePreRulesServiceImpl paymentSourcePreRulesService;

```

Comment:
Unless you explicitly need to reference the Impl class directly, I would suggest adjusting this variable to reference the interface instead.